src/org-manager/containers/organisation-list/organisation-list.component.ts

```diff
@@ -42,7 +44,10 @@ export abstract class OrganisationListComponent implements OnInit {
   }
 
   private doFiltering(): void {
-    if (this.orgs && this.searchString) {
+    const resetStatus = this.route.snapshot.queryParams.reset;
```

Comment:
Might be worth adding a comment above this to explain the reason you are temporarily putting this code in.

Reply:
comments added